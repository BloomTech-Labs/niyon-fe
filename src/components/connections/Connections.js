import React from 'react';
import './styles.scss';

function Connections(props) {
  return (
    <div className='connections' data-test="connections">
      <div data-test="second-wrapper">
        <h1 className='c' data-test="my-connections">My Connections</h1>
        <h1 className='c' data-test="request-connections">Connection Requests</h1>
        <h1 className='c' data-test="recommended-connections">Recommended Connections</h1>
      </div>
    </div>
  );
}

export default Connections;