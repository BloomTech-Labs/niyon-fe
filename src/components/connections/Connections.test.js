import React from 'react';
import { shallow } from 'enzyme';
import Connections from './Connections'
import findByTestAttr from '../../tests/utils';

const setUp = (props={}) => {
  const wrapper = shallow(<Connections { ...props } />);
  return wrapper;
}

describe('<Connections /> component testing', () => {
    let component;
    beforeEach(() => {
      component = setUp();
    });
    it('should render <Connections /> component correctly', () => {
        const connectionsWrapper = findByTestAttr(component, 'connections');
        expect(connectionsWrapper.length).toBe(1);
    });

    it('should render a second container <Div /> correctly', () => {
         const secondWrapper = findByTestAttr(component, 'second-wrapper');
         expect(secondWrapper.exists()).toBe(true);
    });
    it('should render <Paper /> component correctly', () => {
      const wrapper = findByTestAttr(component, 'paper');
        //  expect(wrapper.find('WithStyles(ForwardRef(Paper))').length).toBe(1);
        expect(wrapper.exists()).toBe(true);
    });    
    
    it('should render mani title in the component', () => {
            const h1 = findByTestAttr(component, 'my-connections');
            expect(h1.length).toBe(1);
    });      
        
});