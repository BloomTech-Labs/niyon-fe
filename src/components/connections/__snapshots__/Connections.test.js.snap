// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Connections /> component testing should match snapshot tests 1`] = `undefined`;
